tests/grouppermissions_tests.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+# This script consists of unittests for report_grouppermissions.py
+import contextlib
+import io
+import unittest
+
+import mysql.connector
+import openpyxl
+import openpyxl.utils.exceptions
+
+from python_scripts.repeatable.report_grouppermissions import *
+
+env_file = find_dotenv(f'.env.test')
```

Comment:
@crugas I know this is a wip, but it concerns me to see any tests pointing to aspace02/any live server.  Can we change this to dev?

Reply:
Good point, changed with the most recent commit to point to .dev.